scripts/download_wheels.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Download all wheels for a pandas version.
+#
+# This script is mostly useful during the release process, when wheels
+# generated by the MacPython repo need to be downloaded locally to then
+# be uploaded to the PyPI.
+#
+# There is no API to access the wheel files, so the script downloads the
+# website, extracts the file urls from the html, and then downloads it
+# one by one to the dist/ directory where they would be generated.
+
+VERSION=$1
+DIST_DIR="$(realpath $(dirname -- $0)/../dist)"
+
+if [ -z $VERSION ]; then
+    printf "Usage:\n\t$0 <version>\n\nWhere <version> is for example 1.5.3"
+    exit 1
+fi
+
+curl "https://anaconda.org/multibuild-wheels-staging/pandas/files?version=${VERSION}" | \
+    grep "href=\"/multibuild-wheels-staging/pandas/${VERSION}" | \
+    sed -r 's/.*<a href="([^"]+\.whl)">.*/\1/g' | \
```

Comment:
Maybe not now, but in the future, might want to expand this to downloading the sdist too when we move to cibuildwheel(hopefully for 2.0).

Reply:
There is no sdist now in https://anaconda.org/multibuild-wheels-staging/pandas/files?version=1.5.3 . We can add it by editing the regex if the new implementation adds it. We can also edit the release instructions at that point to not build the sdist locally.